packages/core/src/AutoTypingsCore.ts

```diff
@@ -54,7 +61,7 @@ export class AutoTypingsCore implements monaco.IDisposable {
       dontAdaptEditorOptions: false,
       dontRefreshModelValueAfterResolvement: false,
       sourceCache: AutoTypingsCore.sharedCache ?? new DummySourceCache(),
-      sourceResolver: new UnpkgSourceResolver(),
+      sourceResolver: resolver,
```

Comment:
There is no reason not to allow users to use custom source resolvers here, just putting `sourceResolver: new JsDelivrSourceResolver(),` will use the jsdelivr resolver by default and use custom resolvers instead if provided in the options field, that is applied in line 68.